python/orca/src/bigdl/orca/learn/pytorch/experimential/mmcv/mmcv_ray_estimator.py

```diff
@@ -96,4 +96,9 @@ def evaluate(self, **kwargs):
         pass
 
     def get_model(self):
-        pass
+        """
+        Returns the learned PyTorch model state dict
+        """
+        state = self.get_state_dict()
+        model_state = state["model"]
+        return model_state
```

Comment:
Pytorch provide a way to [save/load the entire model](https://pytorch.org/tutorials/beginner/saving_loading_models.html#save-load-entire-model). but there is a disadvantage

> the serialized data is bound to the specific classes and the exact directory structure used when the model is saved. The reason for this is because pickle does not save the model class itself. Rather, it saves a path to the file containing the class

so Model class must be defined somewhere, but mmcv estimator do not have the Model class, so we can not simply use torch.save/load to get the entire model.

And Pytorch recommends to save the model state dict instead of the entire model. so later in mmcv and pytorch estimator, the get_model method will unify to return the model state dict.